src/components/help/help.spec.js

```diff
@@ -2,15 +2,11 @@ import React from 'react';
 import { act } from 'react-dom/test-utils';
 import 'jest-styled-components';
```

Comment:
this can be removed

Reply:
Cheers for spotting that. It's now been removed.